components/announcement_bar/overage_users_banner/index.tsx

```diff
@@ -0,0 +1,110 @@
+    const user = useSelector(getCurrentUser);
+    const license = useSelector(getLicense);
+    const seatsPurchased = parseInt(license.Users, 10);
+    const isCloud = useSelector(isCurrentLicenseCloud);
+    const getPreferencesCategory = makeGetCategory();
+    const currentUser = useSelector((state: GlobalState) => getCurrentUser(state));
+    const overagePreferences = useSelector((state: GlobalState) => getPreferencesCategory(state, Preferences.OVERAGE_USERS_BANNER));
+    const activeUsers = ((stats || {})[StatTypes.TOTAL_USERS]) as number || 0;
+    const {
+        isBetween5PercerntAnd10PercentPurchasedSeats,
+        isOver10PercerntPurchasedSeats,
+    } = calculateOverageUserActivated({
+        activeUsers,
+        seatsPurchased,
+    });
+    const prefixPreferences = isOver10PercerntPurchasedSeats ? 'error' : 'warn';
+    const preferenceName = `${prefixPreferences}_overage_seats_${seatsPurchased}`;
+
+    const overageByUsers = activeUsers - seatsPurchased;
+
+    const isOverageState = isBetween5PercerntAnd10PercentPurchasedSeats || isOver10PercerntPurchasedSeats;
+
+    if (!isAdmin || !isOverageState || isCloud || adminHasWatchIt({isWarningBanner: isBetween5PercerntAnd10PercentPurchasedSeats, overagePreferences, preferenceName})) {
+        return null;
+    }
+
+    const handleClose = () => {
+        dispatch(savePreferences(currentUser.id, [{
+            category: Preferences.OVERAGE_USERS_BANNER,
+            name: preferenceName,
+            user_id: currentUser.id,
+            value: 'Overage users banner watched',
+        }]));
+    };
+
+    const handleClick = async (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+        e.preventDefault();
+        window.open(LicenseLinks.CONTACT_SALES, '_blank');
+    };
+
+    const message = (
+        <FormattedMessage
+            id='licensingPage.overageUsersBanner.text'
+            defaultMessage='Your workspace user count has exceeded your paid license seat count by {seats, number} {seats, plural, one {seat} other {seats}}. Purchase additional seats to remain compliant.'
+            values={{
+                seats: overageByUsers,
+            }}
+        />);
+
+    const cta = formatMessage({
+        id: 'licensingPage.overageUsersBanner.cta',
+        defaultMessage: 'Contact Sales',
+    });
+
+    return (
+        <AnnouncementBar
+            type={isBetween5PercerntAnd10PercentPurchasedSeats ? AnnouncementBarTypes.ADVISOR : AnnouncementBarTypes.CRITICAL}
+            showCloseButton={isBetween5PercerntAnd10PercentPurchasedSeats}
+            onButtonClick={handleClick}
+            modalButtonText={cta}
+            modalButtonDefaultText={cta}
+            message={message}
+            showLinkAsButton={true}
+            isTallBanner={true}
+            icon={<i className='icon icon-alert-outline'/>}
+            handleClose={handleClose}
+            showCTA={license.Email === user.email}
```

Comment:
Why are we limiting the purchasing admin to Contact Sales? May times the emails used in the license are things like `admin@mattermost.com` and don't match a real admin user

Reply:
If we're going to have a lot of false negatives, I will remove it then.

Reply:
Well I was just wondering if this was part of the requirements in the design or where this came from?

Reply:
This check came from the new design for the global banner; reading the document attached to the epic, I can't see any requirements.